release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -227,6 +227,25 @@ If you have already upgraded your cluster to {product-title} {product-version} a
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.
 
+[id="ocp-4-16-networking-sdn-ovnk-live-migration_{context}"]
+==== Live migration to the OVN-Kubernetes network plugin
+
+Previously, when migrating from OpenShift SDN to OVN-Kubernetes, the only available option was an _offline_ migration method. This process included some downtime, during which clusters were unreachable. 
+
+With this update, a _live_ migration method has been introduced. The live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}, {product-dedicated}, {product-rosa}, and Azure Red Hat OpenShift deployment types. It is not available for HyperShift deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
+
+* Continuous service availability
+
+* Minimized downtime
+
+* Automatic node rebooting
+
+* Seamless transition from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin
+
+Migration to OVN-Kubernetes is intended to be a one-way process.
+
+For more information, see. . .
```

Comment:
@stevsmit Should you include a web link for more information here?

Reply:
@weliang1 yes. I can't include a web link to the content until the feature PR is merged: https://github.com/openshift/openshift-docs/pull/76672